scripts/send-comment.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import requests
+
+
+def get_owner_repo(repo_url):
+    pos = repo_url.rfind('/', 0, repo_url.rfind('/'))
+    return repo_url[pos+1:]
```

Comment:
The above method needs some tweaking.  When I run the pr action, i specify my repo as `http://foo/bar.git`:

```bash
jdo cwr/0 cwr-charm-pr repo=https://github.com/kwmonroe/layer-ubuntu-devenv.git charm-name=blah blah blah
```

The `.git` in the `repo` field throws off the github api:

```python
>>> repo_url = 'https://github.com/kwmonroe/layer-ubuntu-devenv.git'
>>> pos = repo_url.rfind('/', 0, repo_url.rfind('/'))
>>> repo_url[pos+1:]
'kwmonroe/layer-ubuntu-devenv.git'
...
>>> url = 'https://api.github.com/repos/kwmonroe/layer-ubuntu-devenv.git/issues/1/comments'
>>> payload = {'body': 'bar'}
>>> requests.post(url, data=json.dumps(payload), headers={'Authorization': 'token {}'.format(token)})
<Response [404]>
```

The easy fix is to of course say "stop putting .git at the end of the repo param!"  However, since github includes it when you hit the "clone or download" button and choose https, I think it's safe to assume others will include it like I did.  Sooooo.... can you tweak this thing to either chop of `.git` or find another (preferably more flexible) way to construct the github api url?